docs/source/dataformat.rst

```diff
@@ -889,6 +1095,60 @@ For example::
       </verse>
     </lyrics>
 
+Line repeat
+^^^^^^^^^^^
+
+In some songs not only the verses but also the lines may be repeated. Repeated verse can be managed
+with the ``<verseOrder>`` tag::
+
+    <verseOrder>v1 v1 c v2 v2 c</verseOrder>
+
+Repeating lines can be described with an optional attribute for lines::
+
+    <lines repeat="2">O my Jesus.</lines>
+
+The value of this attribute schould be 2 or a larger integer.
+
+Lyrics projectors and processors can display the above example like this::
+
+    𝄆 O my Jesus. 𝄇×2
+
+Or simple::
+
+    O my Jesus.
+    O my Jesus.
+
+Instrumental parts
+^^^^^^^^^^^^^^^^^^
+
+In some songs there are parts without lyrics, instrumental sections or vamps.
```

Comment:
I'm not sure what a vamp is?

Reply:
Vamp: https://en.wikipedia.org/wiki/Ostinato#Vamp It is a common music related word in Hungary, but if it isn't widely known, we can delete it.

Reply:
I would remove it. I would just add 'etc.' instead. In my language it is not known. Is it equivalent to EN word accompaniment?

Reply:
OK, Thanks. "Vamp" is removed from source and replaced with "etc.". 
(Yes, vamp is a repeating musical section.)